tensorflow/core/common_runtime/bfc_allocator.cc

```diff
@@ -229,10 +229,8 @@ void* BFCAllocator::AllocateRaw(size_t unused_alignment, size_t num_bytes,
 }
 
 // static
-size_t BFCAllocator::RoundedBytes(size_t bytes) {
-  size_t rounded_bytes =
-      (kMinAllocationSize *
-       ((bytes + kMinAllocationSize - 1) / kMinAllocationSize));
+inline size_t BFCAllocator::RoundedBytes(size_t bytes) {
```

Comment:
The `inline` here is causing link-time failures when building "//tensorflow/core:common_runtime_gpu_gpu_bfc_allocator_test".

Reply:
I remove it